### ISSUE TITLE
The Newest Nightly Version is Not Working For Me.

### ISSUE THREAD

**Issue by bip3r** [sqlitebrowser/sqlitebrowser#1738]
The nightly version is not working for me.
--------------------

#### What did you do?
Changed my Windows settings to dark mode.
Downloaded and installed the nightly version from here:
https://nightlies.sqlitebrowser.org/latest/
Reset the settings to default.
Restarted the software.

#### What did you expect to see?
The dark version.

#### What did you see instead?
The regular version.

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [x ] Windows: ( _version:Windows 10 pro education )
- [ ] Linux: ( _distro:_ ___ )
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [x ] 3.11.0
- [ ] 3.11.0-alpha1 or 3.11.0-beta*
- [ ] 3.10.1
- [ ] Other: ___

#### Did you also

- [x ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ x] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20
![image](https://user-images.githubusercontent.com/46452395/52527420-0aa6c780-2cd1-11e9-8286-29c4230b750b.png)
![image](https://user-images.githubusercontent.com/46452395/52527436-2d38e080-2cd1-11e9-8dce-f5d6ebc8aa76.png)

**Comment by justinclift** [comment]
Thanks for reporting this @bip3r.  Would you be ok to include a screenshot of the new nightly version, showing it's not working state?

It helps for when we're looking at things, trying to figure out what's going wrong. :smile:

**Comment by mgrojo** [comment]
I seems that Qt doesn't support the Windows 10 themes: https://bugreports.qt.io/browse/QTBUG-72028

We'll have to wait for that bug report to be solved. There is another approach in cofiguring our own dark mode stylesheet through a preference setting. If someone wants to contribute, that would be a good starter project.

**Comment by justinclift** [comment]
Good find @mgrojo.  Just added a comment to that issue about it affecting us too, which hopefully helps it get looked at.  eg it's something that users are affected by

**Comment by bip3r** [comment]
@justinclift  what do you mean?

**Comment by justinclift** [comment]
@bip3r We use a cross-platform tool kit called "[Qt](https://www.qt.io)".  Doing that lets our program work on Windows, macOS, Linux (etc).

It looks like Qt itself has a dark mode problem on Windows 10, so for this to be fixed... we'll probably need to wait for them to get it fixed first.  **\*sigh\*** :frowning:

That's not fantastic.

**Comment by sky5walk** [comment]
Nightly v190311 fixes my issues with Dark Theme while on Windows 10 Pro. 👍
The Scintilla gadget used for the cell editor now respects Dark Theme and the text is visible.

**Comment by sky5walk** [comment]
Ok, getting really close with the Dark Theme 😄 
```
NITPICK:  While browsing either cell editor or SQL text editor, 
          the cursor becomes invisible near any open/close brackets: 
          (parentheses)  [brackets]   {braces}
```
The braces are highlighted in the same color as the cursor.
Please add an option to change highlight Background/Foreground color of all OPEN/CLOSE brackets.

**Comment by justinclift** [comment]
Interesting.  The Scintilla bit looks to be useful in some regards... pity about the lack of RTL support though. :frowning:

**Comment by sky5walk** [comment]
Yes, Scintilla was created for programmers editing text. RTL support is quoted as experimental. I would add optional dropdown selection for RTL text to be shown in a different gadget.

**Comment by justinclift** [comment]
> I would add optional dropdown selection for RTL text to be shown in a different gadget.

Oh, interesting idea.  Having a different widget render RTL text **might** work ok enough for now, until RTL support in QScintilla actually works.

**Comment by mgrojo** [comment]
> > I would add optional dropdown selection for RTL text to be shown in a different gadget.
> 
> Oh, interesting idea. Having a different widget render RTL text **might** work ok enough for now, until RTL support in QScintilla actually works.

Not the best solution in terms of usability and source maintainability, but if we don't find a better solution, should be done before releasing.
